docs/web/howto/client.rst

```diff
@@ -331,74 +331,85 @@ This function returns a ``Deferred`` that fires with the body after the request
 
 
 
-HTTP over SSL
-~~~~~~~~~~~~~
-
-
-    
-
-Everything you've read so far applies whether the scheme of the request
-URI is *HTTP* or *HTTPS* .  However, to control the SSL
-negotiation performed when an *HTTPS* URI is requested, there's
-one extra object to pay attention to: the SSL context factory.
-
-
-
-
-
+Customizing your HTTPS Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-``Agent`` 's constructor takes an optional second argument, a context factory.
-Generally you don't want to create the context factory yourself: it's easy to get wrong.
-Instead, Twisted provides helpers for building correctly configured ones.
-For example, you can use :api:`twisted.internet.ssl.optionsForClientTLS <optionsForClientTLS>`, which takes care of all hostname certificate verification for you.
+Everything you've read so far applies whether the scheme of the request URI is *HTTP* or *HTTPS* .
 
+By default, since version 15.0.0 (if you're using an earlier version, please upgrade, as this is a critical security feature!), Twisted will automatically validate ``https`` URLs against your platform's trust store to the best of its ability.
+Be sure to ``pip install twisted[tls]`` in order to get all the dependencies necessary to do TLS properly, but beyond that you should not need to do much.
 
+However, for some uses, you may need to customize Agent's use of HTTPS; for example, to provide a client certificate, or to use a custom certificate authority for an internal network.
 
+Here, we're just going to show you how to inject the relevant TLS configuration into an Agent, so we'll use the simplest possible example, rather than a more useful but complex one.
 
+``Agent``'s constructor takes an optional second argument, which allows you to customize its behavior with respect to HTTPS.
```

Comment:
As far as I can tell, your comment here is just backwards.  "which allows you to customize its behavior" is (I believe) a restrictive phrase, the closest analog of which I can find in this list of examples https://www.grammarly.com/blog/comma-before-which/ is "Rob tripped over his shoes, which he had left lying in the middle of the floor" and "Everyone loved Robin’s video, which she had filmed in her garage", both of which are listed as _requiring_ a comma (they don't even seem to think it's optional).  So I'm going to ignore it. 

In the future, if you're going to provide grammar rule feedback like this, can you provide some a citation the author can confirm their understanding of the rule you're trying to suggest adherence to?  I have to admit I'm not _confident_ that your understanding of the rules here is wrong and mine is right, but my searching isn't turning up counterexamples.  If it's a stylebook thing, having a link also simplifies future feedback for other authors, for consistency.